### DISCUSSION TITLE
Downloader not working on albums

### DISCUSSION THREAD

**Discussion by krislicoque** [SoCuul/SCInsta#116] [Support] [answered]
When trying to download an image that is part of an album, it shows up like this:

![IMG_6923](https://github.com/user-attachments/assets/932c7511-bed6-493d-aa2b-7e05ba6c1df7)

**Comment by SoCuul** [comment, marked as answer]
As mentioned in the readme, the media downloading features are unfortunately not currently working. I'm hopeful to revamp them in a future version.
